components/indicators/IndicatorList.tsx

```diff
@@ -210,9 +210,10 @@ const getFirstUsableCategoryType = (categoryTypes, indicators) =>
 interface Props {
   title: string;
   leadContent: IndicatorListPage['leadContent'];
+  displayInsights: boolean;
```

Comment:
According to the generated types this could be undefined. Should we make this optional and default to `false` for backwards compatibility?

Reply:
Hmm. That is a problem in the type generation since in the Django backend it cannot be null/undefined and in the GraphQL schema it's a Boolean. But still, I can make the change.

Reply:
Fixed in https://github.com/kausaltech/kausal-watch-ui/pull/170/commits/71122a2b54c67d745fbc2f8e40419f816a330572